Packs/FeedCyrenThreatInDepth/README.md

```diff
@@ -27,11 +27,11 @@ Threat InDepth's timely, correlated, and contextualized intelligence helps enter
 
 ![Cyren GlobalView Cloud](https://raw.githubusercontent.com/demisto/content/master/Packs/FeedCyrenThreatInDepth/doc_files/Cyren_GlobalViewCloud_V2.png)
 
-## Feeds included in Cyren Threat InDepth content pack
+If you don't have an API key already, you should subscribe to one or more of these Content Packs:
 
-The [Cyren Threat InDepth content pack](#/market/browse/FeedCyrenThreatInDepth) includes access to these streams of indicators:
-
-- IP Reputation Intelligence
-- Phishing & Fraud URL Intelligence
-- Malware URL Intelligence
-- Malware File Intelligence
+- [Bundle: Threat Intelligence Feeds](#/market/browse/CyrenThreatInDepthFullBundle)
+- [Bundle: Malware Intelligence Feeds](#/market/browse/CyrenThreatInDepthMalwareBundle)
```

Comment:
Those are broken links, are you sure that's the correct links?

Reply:
We have two pending premium packs pending which are supposed to be linked from here.